template/{% if documentation == 'sphinx' %}docs{% endif %}/conf.py.jinja

```diff
@@ -44,8 +63,31 @@ language = "en"
 
 # -- Options for extensions ----------------------------------------------------
 # https://myst-parser.readthedocs.io/en/latest/syntax/optional.html
-myst_enable_extensions = ["html_image"]
+# -- Options for myst markdown formatting
+myst_enable_extensions = [
+    "html_image",
+    "colon_fence",
+    "deflist",
+    "attrs_inline",
+]
+
+myst_heading_anchors = 3
+myst_footnote_transition = False
+
+
+templates_path = ["_templates"]
+exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
+
+#--------- setup autoapi defaults for your api docs ---------------
 
+# AutoAPI configuration
+autoapi_type = "python"
+# point AutoAPI at your package sources; adjust if using src layout
+autoapi_dirs = ["../src"]
+# Don't let AutoAPI automatically insert a toctree (avoid duplicates)
+autoapi_add_toctree = False
```

Comment:
Autoapi magically creates documentation. It' just a really nice, easy starting place. and we can redirect people to autodoc if they want more custom builds -- maybe the scientific Python guide covers that?